packages/neo-one-cli/src/utils/reportError.ts

```diff
@@ -0,0 +1,22 @@
+import { CLIArgs } from '@neo-one/server-plugin';
+import * as fs from 'fs-extra';
+
+interface ReportErrorOpts {
+  readonly logFilePath: string;
+  readonly monitor: CLIArgs['monitor'];
+}
+
+export const reportError = async ({ logFilePath, monitor }: ReportErrorOpts) => {
+  try {
+    const serverLogFile = `${logFilePath} server`;
```

Comment:
What's going on here? This isn't a valid path.

Reply:
Testing.
This hadn't yet been exercised, it's been a few days, I think it's missing the server.log portion.
This PR was submitted to get preliminary feedback on approach.